src/turn_integrator.py

```diff
@@ -3,20 +3,21 @@
 import requests
 import json
 
-### Load and evaluate the cedentials from the turn_config.json file
-def load_credentials(file_name, line_name):
+## SETUP
+## Load and evaluate the cedentials from the turn_config.json file
+def load_credentials(file_name: str, line_name: str) -> str:
     with open(file_name, 'r') as file:
         turn_config = json.load(file)
 
     return turn_config['lines'][line_name]
 
-def eval_credentials(config_json):
+def eval_credentials(config_json: json) -> str:
```

Comment:
I appreciate the adding of return types. I think it is something that makes Python code more readable.

Reply:
Can we add some jsdocs style comments to these methods, to communicate their purpose and what exceptions they may raise, so that the consumer can ensure they handle the scenarios.

Reply:
These [Docstrings](https://peps.python.org/pep-0257/) are porobably the best use. I like the Google conventions in 3.8 of their [style guide](https://google.github.io/styleguide/pyguide.html)